### PR TITLE
Branding and Customization - Updates per Jesus and additional suggestions

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/branding-and-customization.adoc [hunk 1]
```diff
@@ -4,98 +4,101 @@ include::_attributes.adoc[]
...
-* *Images*: Set the site's *Company Logo* and the login page's *Background Image*.
-* *Fonts*: The site's fonts include the *Primary Font* and *Header Fonts*. You can also set the site's *Text Case*.
-* *Theme Settings*: The site's theme settings include *Show header*, *Fix the header's position*, *Show hero* to enable the site's hero image, *Start hero under header*, and *Set max page width* in pixels.
+Select the *Components* lightning bolt icon in the top left navigation to display the component palette. 
+The component palette includes Salesforce Community Cloud and ACM-specific components. 
+Drag and drop the desired components from the component palette to the community page. Each 
+component has its own configuration options that you can then access by selecting the component on the page.
 
-== Theme Navigation Menu
+For additional component information, refer to the Salesforce Community Cloud and ACM product documentation.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/branding-and-customization.adoc]: please add links here to: 
community cloud components docs: https://help.salesforce.com/articleView?id=rss_component_koa_kokua_components.htm&type=5
ACM components: link to the corresponding page in our docs

**Review by jdeoliveira — APPROVED** [review]

File: modules/ROOT/pages/branding-and-customization.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/branding-and-customization.adoc]: I've updated this  - thank you! @valkyrie69 - can you please check the relative link syntax?

File: modules/ROOT/pages/branding-and-customization.adoc [hunk 2]
```diff
@@ -4,98 +4,103 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Use the Community Builder to customize your community and add branding, images and logos.
+Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
 
-In the ACM administrator interface, select the *Community Builder* link.
+Select *Community Builder* in the ACM administrator interface. By default, 
+the Community Builder displays the *Home* page. You can select the *Home* drop-down list arrow to display a list
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/branding-and-customization.adoc]: This is the **Pages** drop-down list, and always displays the title of the current page, which is sometimes **Home** but not always.

File: modules/ROOT/pages/branding-and-customization.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/branding-and-customization.adoc]: The relative link syntax is correct, but the relative link should be to the components page and not the index page. Fixed.

That Salesforce link is for Koa and Kokua components, and that page has a note saying those were replaced in Summer '17 by the Lightning Components, and I know ACM uses Lightning Components, so I changed the link to the Lightning Components page.

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**